bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/catalog/Horizons.scala

```diff
@@ -56,10 +56,10 @@ class Horizons private (/*host: String, port: Int,*/ site: Site, start: Date, en
           // Ambiguous
           case MUTLIPLE_ANSWER =>
             val table = hr.getResultsTable
-            val header = table.getHeader.asScala.toSeq
+            val header = table.getHeader.asScala
 
             val options = for {
-              row <- table.getResults.asScala.map(_.asScala.toSeq).toSeq
+              row <- table.getResults.asScala.map(_.asScala.toSeq)
```

Comment:
IDEA reported that the `toSeq` calls here were redundant, so I removed them for cleanness.